### ISSUE TITLE
GetStats -> Nintendo Switch player

### ISSUE THREAD

**Issue by hadouken111** [Tustin/fortnite-php#16]
Hey,

is there a way to get the stats of a switch/IOS/Android profile?

**Comment by monkeydioude** [comment]
@hadouken111 I see you firing questions in this repo and in the javascript api repo. Can you sometimes dig and try to help communities instead of only profiting ?

**Comment by hadouken111** [comment]
I simply linked the question that was asked, so I'm not the only one asking for help.
If I can help, I will do it with pleasure ;)

**Comment by monkeydioude** [comment]
Obviously, helping people is one of the main reasons Open Software exists. Exchange too.

I'll just admit that I reacted a bit too fast on this one :)

But I'll be watching you :eyes:

**Comment by monkeydioude** [comment]
Anyone playing on Switch ? I can try something with my own API. I tried with username from the reddit Fortnite Switch, could not find any info about them but have no certainty their reddit username is the same as IG nickname.

**Comment by hadouken111** [comment]
no worries dude :)

**Comment by RobertoGraham** [comment]
This is the response for user "Kronosbeen" who definitely plays on Switch:
`[{"name":"br_score_pc_m0_p10","value":10554,"window":"weekly","ownerType":1},{"name":"br_score_pc_m0_p9","value":330,"window":"weekly","ownerType":1},{"name":"br_matchesplayed_pc_m0_p10","value":77,"window":"weekly","ownerType":1},{"name":"br_score_pc_m0_p2","value":5285,"window":"weekly","ownerType":1},{"name":"br_kills_pc_m0_p2","value":16,"window":"weekly","ownerType":1},{"name":"br_matchesplayed_pc_m0_p2","value":52,"window":"weekly","ownerType":1},{"name":"br_matchesplayed_pc_m0_p9","value":2,"window":"weekly","ownerType":1},{"name":"br_placetop12_pc_m0_p10","value":21,"window":"weekly","ownerType":1},{"name":"br_placetop10_pc_m0_p2","value":3,"window":"weekly","ownerType":1},{"name":"br_placetop25_pc_m0_p2","value":12,"window":"weekly","ownerType":1},{"name":"br_placetop5_pc_m0_p10","value":10,"window":"weekly","ownerType":1},{"name":"br_kills_pc_m0_p10","value":21,"window":"weekly","ownerType":1},{"name":"br_placetop6_pc_m0_p9","value":1,"window":"weekly","ownerType":1}]`

Switch Fortnite has no stats tab so it's unlikely they're available on any endpoint

**Comment by Tustin** [comment]
iOS doesn't seem to have stats available (at least when I last checked). However, I can look to see if the Switch has stats and if so, I'll add it.

**Comment by Tustin** [comment]
Switch doesn't seem to have any stats yet, but I'll leave this open as they might in the future.
